kaitai-struct-visualizer.gemspec

```diff
@@ -1,9 +1,9 @@
-# -*- encoding: utf-8 -*-
+# frozen_string_literal: true
 
-require File.expand_path("../lib/kaitai/struct/visualizer/version", __FILE__)
+require File.expand_path('lib/kaitai/struct/visualizer/version', __dir__)
```

Comment:
Just want to document that `__dir__` doesn't seem to depend on the current working dir and should therefore be safe and what's actually done in the code before:

    File.dirname(File.realpath(__FILE__))

https://stackoverflow.com/a/15190725/2055163
https://bugs.ruby-lang.org/issues/7975
https://web.archive.org/web/20100414044710/http://www.ruby-forum.com/topic/193674